core-bundle/src/Resources/contao/dca/tl_page.php

```diff
@@ -231,24 +228,23 @@
 			'search'                  => true,
 			'inputType'               => 'text',
 			'eval'                    => array('rgxp'=>'folderalias', 'doNotCopy'=>true, 'maxlength'=>255, 'tl_class'=>'w50 clr'),
-			'save_callback' => array
-			(
-				array('tl_page', 'generateAlias')
-			),
 			'sql'                     => "varchar(255) BINARY NOT NULL default ''"
 		),
+		'parameters' => array
```

Comment:
Can you give me a hint where the `parameters` are applied? I couldn't find it out on my own so far.

Reply:
This seems like a left-over field that should be removed.

Reply:
It is not a left-over. It is the source for custom URL parameters, and implemented in https://github.com/contao/contao/pull/1516/files#diff-160cbdb94b54d500f3a4df0f81fc8f47R64. There's just currently no page type yet that supports it.